lighthouse-core/audits/byte-efficiency/bundlephobia-database.json

```diff
@@ -0,0 +1,2999 @@
+{
```

Comment:
if this is from bundlephobia's repo, we need to follow the steps under https://opensource.google/docs/releasing/preparing/#third-party-components (a `bundlephobia/` directory under `/third-party/` with a `LICENSE` file and this inside, ideally with a clean commit and a readme pointing out when it was cloned before we make any alterations)

Reply:
oh, sorry, just saw the generating script. So that's a little different, but should still live under `third-party/bundlephobia/` with the LICENSE alongside :)

Reply:
what license do you hope to have here @brendankenny ? there's no code from the repository here and the shape of the data is not the same either, nor does bundlephobia have any claim to it :/

the only thing reproduced here is data from the bundlephobia *API*, namely the gzipped stats, description and repository (which are in fact mirrored from the repos themselves and would be governed by the licenses of each individual repo). 

is the hope to reproduce every license file of every alternative library we reference? AFAICT if we removed the descriptions that we don't use, the ability to own any intellectual property over what's in this database would be questionable at best. bundlephobia doesn't actually seem to have a terms of use policy I could find on their site, so I'm guessing they're in the same boat I'm in which is, no one can come after you for mirroring a list of URLs + a stat on the files that were published publicly under an open license. 

we should probably have the script double check the license of each alternative we're recommending though to make sure it's compatible and open 🤔

Reply:
> we should probably have the script double check the license of each alternative we're recommending though to make sure it's compatible and open 🤔

feature creeeeeeeeeep

Let's leave license compat checks until someone complains about our suggestions being annoying and impossible because of licences :) I'd even say that BP should remove a dependency if their license is not friendly.

Reply:
To be clear, I don't think this is necessary. I'm proposing this as an alternative to checking in dozens of unrelated license files.

Reply:
lol let's all chill a bit, I obviously didn't read carefully enough. I was assuming based on the file name that this was the content that's in `library-suggestions.js`, while this is actually npm registry info.

I don't want to speak too quickly in case I'm mistaken again :) but if `library-suggestions.js` is the info from bundlephobia, then what I said above may apply to that.